packages/node_modules/@node-red/nodes/core/network/10-mqtt.html

```diff
@@ -10,6 +10,54 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
+<style>
+    .form-row label:first-child {
```

Comment:
Just for future reference as I'll tidy it up in merge, but need to be careful not to apply global styles like this. Needs to be more carefully scoped to this node.

Reply:
oops on the global style. In my head it only applied to `.form-row` and only for this node.  After reading your commend and using a bit of brain power, I realise the CSS can affect other nodes when their form is opened.